docs/watcher/README.md

```diff
@@ -11,10 +11,10 @@ The Watcher currently supports the following types:
 
 ## Result Grouping
 
-The Watcher uses annotations to automatically detect and group related Records into the same Result. The following annotations are recognized (in order of precedence):
+The Watcher uses Object data to automatically detect and group related Records into the same Result. The following data is checked (in order of precedence):
 
-- `results.tekton.dev/result`
-- `triggers.tekton.dev/triggers-eventid`
-- `tekton.dev/pipelineRun`
+- `results.tekton.dev/result` annotation. This should correspond to the full `Result.name` identifier (e.g. `foo/results/bar`).
+- `triggers.tekton.dev/triggers-eventid` annotation (this is generated from Objects created via [Tekton Triggers](https://github.com/tektoncd/triggers))
```

Comment:
Triggers adds this as a label not an annotation.

Reply:
Good catch! Fixed.